src/sentry/interfaces/schemas.py

```diff
@@ -293,7 +293,7 @@ def apierror(message="Invalid data"):
         },
         'logger': {
             'type': 'string',
-            'pattern': r'^[a-zA-Z0-9_\.:-]+$',
+            'pattern': r'^[a-zA-Z0-9_\.:-\$]+$',
```

Comment:
I don't think you technically have to escape a `$` inside a character class, but it also works this way.

Reply:
Oh right, it's unambiguous as a single char I guess. :thunking-face: